src/java/com/puppetlabs/puppetserver/jruby/ScriptingContainer.java

```diff
@@ -0,0 +1,35 @@
+package com.puppetlabs.puppetserver.jruby;
+
+import org.jruby.embed.LocalContextScope;
+import org.jruby.embed.LocalVariableBehavior;
+
+/**
+ * An extension of the JRuby ScriptingContainer class which is
+ * slightly easier to use from Clojure
+ */
+public class ScriptingContainer
+        extends org.jruby.embed.ScriptingContainer {
+    public ScriptingContainer(LocalContextScope scope) {
+        super(scope);
+    }
+    public ScriptingContainer(LocalContextScope scope, LocalVariableBehavior behavior) {
+        super(scope, behavior);
+    }
+
+    /**
+     * This method delegates to a specific signature of #callMethod from the
```

Comment:
Yeah, that all makes sense.  Was just curious if something about the changes on this PR caused this problem where we weren't seeing it before since the code changes at the Clojure level didn't seem obviously different.  I'm good with the changes you made here.